db/schema.sql

```diff
@@ -5,7 +5,7 @@ CREATE TABLE IF NOT EXISTS cohorts (
 );
 
 CREATE TABLE IF NOT EXISTS students (
-  id SERIAL PRIMARY KEY,
+  enrollment_id integer PRIMARY KEY,
```

Comment:
This needs to be in a join table, `enrollments`, not an ID in the student record.  Many students are enrolled in more than one cohort.  If you'd like to make this a cache column for performance reasons, you can make it current_enrollment_id which will be updated any time a student is enrolled or deferred to a cohort.

enrollments
-------------------
id
student_id
cohort_id